### PR TITLE
Removed path prefix for gtg endpoint, logged input params and updated…

### PR DESCRIPTION
**Pull request by davidbalazs93**
Changes in this PR:
 - removed path prefix for gtg endpoint, meaning that from now on the gtg endpoint will be accessible by __gtg instead of <pathPrefix>/__gtg (__health/__gtg)
- logged input params
- updated documentaion
- added some more tests
- added parallel severity checking

### CODE REVIEW

**Comment by davidbalazs93** [comment]
do not review this until tests are fixed

File: README.md [hunk 1]
```diff
@@ -35,35 +36,46 @@ To run the service locally, you will need to run the following commands first to
...
     - params:
        - `service-name` - The service to be updated.
- * `enable-category` - Enables a category. This is used for sticky categories which are unhealthy.
+ * `<pathPrefix>/enable-category` - Enables a category. This is used for sticky categories which are unhealthy.
     - params:
        - `category-name` - The category to be enabled.
- * `disable-category` - Disables a category. This is useful when doing a failover.
+ * `<pathPrefix>/disable-category` - Disables a category. This is useful when doing a failover.
     - params:
        - `category-name` - The category to be disabled.
```
Comment (tamas-molnar) [inline README.md]: Would it be worth documenting how to configure categories, default values etc?

Comment (davidbalazs93) [inline README.md]: yep, did it, thanks :+1:

File: README.md [hunk 2]
```diff
@@ -35,35 +36,46 @@ To run the service locally, you will need to run the following commands first to
  The list of functionalities that can be used outside of the cluster are:
   * Add/Remove acknowledge
   * Enable/Disable sticky categories
+## How to configure services for aggregate-healthcheck
+ For a service to be taken into consideration by aggregate-healthcheck it needs to have the following:
+ * The Kubernetes service should have __hasHealthcheck: "true"__ label.
+ * The container should have Kubernetes `readinessProbe` configured to check the `__gtg` endpoint of the app
+ * The app should have `__gtg` and `__health` endpoints.
+ * Optionally the Kubernetes service can have:
```
Comment (davidbalazs93) [inline README.md]: yep, thanks :+1:

**Pull request merged**